Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/orientation.h

```diff
@@ -40,30 +40,34 @@ namespace CGAL {
 namespace Polygon_mesh_processing {
 
 namespace internal{
-  template <class Less_xyz, class VPmap>
-  struct Compare_vertex_points_xyz_3{
-    Less_xyz less;
+
+  template <class GT, class VPmap>
+  struct Compare_vertex_points_z_3
+  {
+    typename GT::Compare_z_3 compare_z;
     VPmap vpmap;
 
-   Compare_vertex_points_xyz_3(VPmap const& vpmap)
-	: vpmap(vpmap){}
+    Compare_vertex_points_z_3(VPmap const& vpmap, const GT& gt)
+      : vpmap(vpmap)
+      , compare_z(gt.compare_z_3_object())
+    {}
 
     typedef bool result_type;
     template <class vertex_descriptor>
     bool operator()(vertex_descriptor v1, vertex_descriptor v2) const
     {
-      return less(get(vpmap, v1), get(vpmap, v2));
+      return CGAL::SMALLER == compare_z(get(vpmap, v1), get(vpmap, v2));
     }
-
   };
 
+
   template<typename Kernel, typename PM, typename NamedParameters>
   bool is_outward_oriented(typename boost::graph_traits<PM>::vertex_descriptor vd,
                            const PM& pmesh,
                            const NamedParameters& np)
   {
-    const typename Kernel::Vector_3&
-      normal_v_min = compute_vertex_normal(vd, pmesh, np);
+    const typename Kernel::Vector_3& normal_v_min
+      = CGAL::Polygon_mesh_processing::compute_vertex_normal(vd, pmesh, np);
```

Comment:
That function `is_outward_oriented(vertex_descriptor, Mesh)` is actually no longer used. It was used by `is_outward_oriented(Mesh)` but it is no longer, since the implementation changed.